### DISCUSSION TITLE
Hi everyone,

### DISCUSSION THREAD

**Discussion by aleka07** [community/community#137388] [GitHub Education]
### Hi everyone,

I applied for the GitHub Student Developer Pack five days ago but haven’t received it yet. The instructions mentioned that if it takes more than five days, I should start a discussion in the GitHub community. So, I'm following up on that guidance here and hope that my request will be noticed and addressed. Thank you for your assistance.

**Comment by davevad93** [comment]
Hi @aleka07 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @aleka07 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
